profile.md

```diff
@@ -0,0 +1,173 @@
+# Open Service Broker API Profile
+
+## Abstract
+
+The [Open Service Broker API specification](spec.md) allows for extensions
+and variations based on the environments in which it is being used; this
+document contains the recommended usage pattern for some of those variants.
+
+## Table of Contents
+
+- [Notations and Terminology](#notations-and-terminology)
+- [Context Object](#context-object)
+  - [Context Object Properties](#context-object-properties)
+  - [Cloud Foundry](#cloud-foundry)
+  - [Kubernetes](#kubernetes)
+
+## Notations and Terminology
+
+### Notational Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to
+be interpreted as described in [RFC 2119]( https://tools.ietf.org/html/rfc2119).
+
+### Terminology
+
+The terminology defined below are defined by the
+[Open Service Broker API specification](spec.md) and are included here
+for convinience. If there are any inconsistencies between their definitions in
+this document and the [Open Service Broker API specification](spec.md), then
+the [Open Service Broker API specification](spec.md) SHALL take precidence.
+
+- *Platform*: The software that will manage the cloud environment into which
+  Applications and Service Brokers are provisioned.  Users will not directly
```

Comment:
That some platforms, either now or in the future, may not call things Applications. Or maybe services will be able to be bound to things like spaces or orgs, rather than apps.